packages/components/organisms/f-header/src/components/_tests/CountrySelector.test.js

```diff
@@ -0,0 +1,169 @@
+import { shallowMount } from '@vue/test-utils';
+
+import CountrySelector from '../CountrySelector.vue';
+
+const $style = {
+    'is-open': 'is-open'
+};
+
+const defaultPropsData = {
+    tabindex: 0
+};
+
+let wrapper;
+
+describe('Country Selector', () => {
+    it('should have "is-open" class when isCountrySelectorOpen is true', async () => {
+        // Arrange
+        wrapper = shallowMount(CountrySelector, {
```

Comment:
Could this be in a `beforeEach` block

Reply:
It probably could, and I'll move it if it gets much more complicated, but I think it's short and simple enough at the moment, and having the `Arrange` within the `it` helps readability, in my opinion.